postgresql-async/src/test/java/com/github/aysnc/sql/db/integration/PostgreSQLConnectionSpec.kt

```diff
@@ -79,7 +79,7 @@ class PostgreSQLConnectionSpec : DatabaseTestHelper() {
             time_column time,
             boolean_column boolean,
             constraint bigserial_column_pkey primary key (bigserial_column)
-          ) with oids"""
+          )"""
```

Comment:
Support for this query syntax appears to have been removed in later version is PostgreSQL: https://www.postgresql.org/message-id/E1gPG3j-0003cy-03%40gemulon.postgresql.org